src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -674,13 +674,12 @@ grpc_error_handle Chttp2ServerListener::Create(
       auto string_address = grpc_sockaddr_to_uri(addr);
       if (!string_address.ok()) {
         return GRPC_ERROR_CREATE_FROM_CPP_STRING(
-            std::string(string_address.status().ToString()));
+            string_address.status().ToString().c_str());
```

Comment:
You do not need to call `c_str()` here.  That will result in unnecessarily constructing a new copy of the returned string.  `GRPC_ERROR_CREATE_FROM_CPP_STRING()` takes a `std::string` as its argument, and that's exactly what `absl::Status::ToString()` returns.